_posts/02-01-01-Code-Style-Guide.md

```diff
@@ -2,60 +2,43 @@
-to code-style, but those that do are [PSR-0][psr0], [PSR-1][psr1], [PSR-2][psr2] and [PSR-4][psr4]. These
-recommendations are merely a set of rules that many projects like Drupal, Zend, Symfony, Laravel, CakePHP, phpBB, AWS SDK,
-FuelPHP, Lithium, etc are adopting. You can use them for your own projects, or continue to use your own
-personal style.
+[Framework Interop Group][fig] এর দ্বারা প্রস্তাবিত এবং অনুমোদিত কিছু নীতিমালা আছে। এর সবগুলোই কোড স্টাইল সম্পর্কিত নয়, যেগুলো এ সম্পর্কিত সেগুলো হল [PSR-0][psr0], [PSR-1][psr1], [PSR-2][psr2] এবং [PSR-4][psr4]। এই নীতিমালাগুলো কেবল কিছু নিয়মের সমন্বয় যেগুলো Drupal, Zend, Symfony, Laravel, CakePHP, phpBB, AWS SDK, FuelPHP, Lithium ইত্যাদি প্রজেক্টও মেনে চলে। যে কেউ ইচ্ছা করলে তার প্রজেক্টে এই নিয়মগুলোও মেনে চলতে পারে অথবা তার নিজস্ব স্টাইলও মেনে চলতে পারে।
 
-Ideally, you should write PHP code that adheres to a known standard. This could be any combination of PSRs, or one
-of the coding standards made by PEAR or Zend. This means other developers can easily read and work with your code, and
-applications that implement the components can have consistency even when working with lots of third-party code.
+কোন একটি প্রচলিত নিয়ম অনুসরণ করে পিএইচপি কোড লিখা উচিৎ। তা হতে পারে কতগুলো PSR এর সমন্বয় অথবা PEAR বা Zend এর তৈরিকৃত কোন মানদণ্ড। অর্থাৎ অন্যান্য ডেভেলপারেরা যেন সহজেই আপনার কোড পড়তে পারে ও এটি নিয়ে কাজ করতে পারে এবং যেসব অ্যাপ্লিকেশন্স কম্পোনেন্টগুলোকে ইমপ্লিমেন্ট করে তাদের মাঝেও যেন সামঞ্জস্যতা থাকে এমনকি যখন অসংখ্য থার্ড-পার্টি কোড নিয়ে কাজ করা হয়।
 
-* [Read about PSR-0][psr0]
-* [Read about PSR-1][psr1]
-* [Read about PSR-2][psr2]
-* [Read about PSR-4][psr4]
-* [Read about PEAR Coding Standards][pear-cs]
-* [Read about Symfony Coding Standards][symfony-cs]
+* [PSR-0 সম্বন্ধে জানুন][psr0]
+* [PSR-1 সম্বন্ধে জানুন][psr1]
+* [PSR-2 সম্বন্ধে জানুন][psr2]
+* [PSR-4 সম্বন্ধে জানুন][psr4]
+* [PEAR কোডিং স্ট্যান্ডার্ডস সম্বন্ধে জানুন][pear-cs]
+* [Symfony কোডিং স্ট্যান্ডার্ডস সম্বন্ধে জানুন][symfony-cs]
 
-You can use [PHP_CodeSniffer][phpcs] to check code against any one of these recommendations, and plugins for text
-editors like [Sublime Text][st-cs] to be given real-time feedback.
+উপরোক্ত নিয়মগুলোর বিপরীতে কোড চেক করার জন্য [PHP_CodeSniffer][phpcs] ব্যবহার করা যেতে পারে এবং টেক্সট এডিটর [Sublime Text][st-cs] এর প্লাগইনস ব্যবহার করা যেতে পারে যেটি রিয়েল-টাইম ফিডব্যাক প্রদর্শন করে।
 
-You can fix the code layout automatically by using one of the following tools:
+নিচের যেকোন একটি টুলস ব্যবহার করে স্বয়ংক্রিয়ভাবে কোড লেআউট ঠিক করা যায়ঃ
 
-- One is the [PHP Coding Standards Fixer][phpcsfixer] which has a very well tested codebase.
-- Also, the [PHP Code Beautifier and Fixer][phpcbf] tool which is included with PHP_CodeSniffer can be used to adjust your code accordingly.
+- একটি হল [PHP Coding Standards Fixer][phpcsfixer] যার কোডবেস খুব ভালোভাবে Tested ।
+- এছাড়াও আছে [PHP Code Beautifier and Fixer][phpcbf] যেটি PHP_CodeSniffer এর সাথেই দেয়া থাকে। এর সাহায্যে কোডকে নিজের পছন্দমত স্টাইল এ সাজানো যায়। .
```

Comment:
The extra fullstop "." at the end of line should be removed.

Reply:
Done :)